gradle/java-library.gradle

```diff
@@ -4,8 +4,8 @@ group = 'org.mockito'
 archivesBaseName = "mockito-" + project.name
 bintrayUpload.enabled = true
 
-sourceCompatibility = 1.6
-targetCompatibility = 1.6
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
```

Comment:
This is a major breaking change. We can not update the Java version of the library to 8 in Mockito 2.

Reply:
Is there an expected timeline for moving to Java 8?

Reply:
That is Mockito 3 and we have no concrete plans for any kind of release date right now.